CMake/SlicerCPackShortcutHelper.nsh

```diff
@@ -0,0 +1,133 @@
+; Copyright Safing ICS Technologies GmbH. Use of this source code is governed by the Apache 2 license that can be found in the LICENSE file.
+; Adapted from https://github.com/safing/nsis-shortcut-properties/blob/b5ede73fc49665449731df5d0f945192329b0ef6/shortcut-properties.nsh
```

Comment:
Can you confirm that it does not require user access elevation (can run the installer as a regular user)?

Is there a minimum required Windows version for this to work? If it works on Windows7 (it does not matter if it displays an error or warning, as long as it can be installed) then it is good. If it requires anything above Windows7 then we need to update the documentation with that requirement (https://slicer.readthedocs.io/en/latest/user_guide/getting_started.html#operating-system-versions).

Reply:
I can confirm that no elevation is required. I could install at my local user with no problem.
Confirmed that it runs on win7 using the demo nsis project from the linked original github project, i.e. tested the nsh script outside the slicer build environment on win7 and it worked.